install/connector/bin/java.policy

```diff
@@ -24,15 +24,15 @@ grant codeBase "file:${com.sun.enterprise.home}/lib/system/-" {
     permission java.security.AllPermission;
 };
 
-grant codeBase "file:${com.sun.enterprise.home}/lib/javaee.jar" {
+grant codeBase "file:${com.sun.enterprise.home}/lib/jakartaee.jar" {
```

Comment:
I'm not sure if this policy file is used somehow, but I am sure this file was renamed. If the policy file is used, this change might fix some test.